### PR TITLE
Fix style in connector module

### PR DESCRIPTION
**Pull request by matthiasbeyer**
This PR cleanes the style in the connector module (the branchname may not be
appropriate here..., please don't mind!).

This PR also introduces a logic-fix, so it is not _entirely_ about style.

@neithernut Please review carefully (the logic-fix), I am (/ we are)
not entirely sure about it,...

### CODE REVIEW

File: src/connection/processor.c [hunk 1]
```diff
@@ -369,7 +368,7 @@ connection_manager_flush_msg(
 
         // flush
         res = ws_connector_flush(&proc->conn);
-    } while (res > 0);
+    } while (res == 0);
```
Comment (manuelmessner) [inline src/connection/processor.c:371]: I would like to hear @neithernut 's opinion about that as well.

Comment (neithernut) [inline src/connection/processor.c:371]: The initial intent was to try flushing stuff as long as writing to the fd doesn't block. However, this was overlooked when working on the `ws_connector_flush()` method.
As it _should_ return `-EAGAIN` if no more data can be written atm, this _should_ be a legetimate fix.

Comment (matthiasbeyer) [inline src/connection/processor.c:371]: Yay, we did something right! :smile:

Comment (xVin) [inline src/connection/processor.c:371]: I'm going to write a fix for the flush method.

**Comment by neithernut** [comment]
ACK.

**Comment by matthiasbeyer** [comment]
@neithernut Feel free to merge after you ACKed (also for all other PRs)!

File: src/connection/processor.c (hunk 1 above)
Comment (neithernut) [inline src/connection/processor.c:371]: This isn't neccessary, as it does exactly what it should now. What I meant when I said that this was overlooked was that I overlook this when I was looking at EOF issues.

**Pull request merged**